tools/integration_tests/concurrent_operations/setup_test.go

```diff
@@ -47,7 +48,7 @@ func TestMain(m *testing.M) {
 
 	// Create common storage client to be used in test.
 	ctx = context.Background()
-	closeStorageClient := client.CreateStorageClientWithTimeOut(&ctx, &storageClient, time.Minute*15)
+	closeStorageClient := client.CreateStorageClientWithTimeOut(&ctx, &storageClient, time.Minute*50)
```

Comment:
Seems like these tests will add significant time to the test execution time. Do you think adding this to our regular test suite will add a lot of value? I see most of the scenarios are overlapping with the existing tests we added for concurrent operations. Shall we instead modify those tests to have an outer wait group to run them concurrently and make it configurable by the developer instead of relying on CPU cores? This will not increase the test execution time and developers can run it for confidence when they are making any core changes.